### PR TITLE
[PWGJE/EMCALJetTasks/UserTasks/]: Add formation time to jet substructure task.

### PR DESCRIPTION
**Pull request by hjbossi**
In addition some small spacing issues were fixed in order to make the code more readable.

### CODE REVIEW

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx [hunk 1]
```diff
@@ -302,6 +304,10 @@ void AliAnalysisTaskNewJetSubstructure::UserCreateOutputObjects() {
     }
 
   fShapesVarNames[22] = "etaJet";
+  if(fDoTime){
+    fShapesVarNames[23] = "tauForm";
+    fShapesVarNames[24] = "tauFormMatch";
+  }
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx:310]: I'm a little rusty on this task, but looking ahead a little bit, I think you're missing `tauDet`, which will be used when you're embedding

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx [hunk 2]
```diff
@@ -1096,12 +1119,13 @@ void AliAnalysisTaskNewJetSubstructure::IterativeParentsPP(AliEmcalJet *fJet, Al
...
-	j2first =j2;
-	*sub2 = j2first;
+        tau_g = tauForm;
+        j1first =j1;
+        *sub1 = j1first;
+        j2first =j2;
+        *sub2 = j2first;
         flagSubjet = 1;
-	ptone = j1.perp();
+	      ptone = j1.perp();
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx:1128]: spacing issue here

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx [hunk 3]
```diff
@@ -1219,13 +1247,14 @@ void AliAnalysisTaskNewJetSubstructure::IterativeParentsMCAverage(
...
-	j2first = j2;
-	*sub2 = j2first;
+	    zg = z;
+      xktg = xkt;
+      Rg = delta_R;
+      tau_g = tauForm;
+      j1first = j1;
+      *sub1 = j1first;
+      j2first = j2;
+      *sub2 = j2first;
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx:1257]: inconsistent spacing here

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx [hunk 4]
```diff
@@ -581,13 +590,14 @@ Bool_t AliAnalysisTaskNewJetSubstructure::FillHistograms() {
           kMatched = 3;
 
         ptMatch = jet3->Pt();
-	leadTrackMatch = jet3->MaxTrackPt();
+	      leadTrackMatch = jet3->MaxTrackPt();
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx:593]: spacing

**Comment by hjbossi** [comment]
@raymondEhlers the new commit should fix your earlier comments - let me know if you see any additional issues!

**Comment by raymondEhlers** [comment]
+1 looks good. sorry, I forgot to check this earlier

**Pull request merged**